chromium_src/third_party/blink/renderer/platform/loader/fetch/resource_fetcher.cc

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "third_party/blink/renderer/platform/loader/fetch/resource_fetcher.h"
+
+#include "third_party/blink/public/common/features.h"
+#include "third_party/blink/renderer/platform/loader/fetch/memory_cache.h"
+
+#define DefaultCacheIdentifier                           \
+  DefaultCacheIdentifier().IsEmpty()                     \
+      ? GetCacheIdentifierIfCrossSiteSubframeOrDefault() \
```

Comment:
how about adding a comment to minimise WTF-effect

Reply:
added more comments in various parts of the PR. the WTF-level should be decreased.